apps/src/lab2/views/components/versionHistory/VersionHistoryDropdown.tsx

```diff
@@ -303,7 +303,7 @@ const VersionHistoryDropdown: React.FunctionComponent<
   // if the user is viewing an old version, then close the dropdown.
   const handleCancel = useCallback(() => {
     // Go back to current version if we are viewing an old version
-    if (!isLatestVersion(selectedVersion)) {
+    if (selectedVersion && !isLatestVersion(selectedVersion)) {
```

Comment:
Nice catch!
Nit - could be addressed in a follow-up. if the user selects the 'Initial version' option for a new project, the 'You are viewing an older version' banner is displayed and then the start over dialog appears after 'Restore' is clicked. Wondering if it would make more sense to just have the Initial Version be 'current' and the 'Restore' button be disabled if there is no version history yet.

https://github.com/user-attachments/assets/0072a0cc-01aa-4c91-9be4-805f583f560c

Reply:
Good idea! Filed [CT-1149](https://codedotorg.atlassian.net/browse/CT-1149) to track.